### PR TITLE
Update Hyrax and pull in new Blacklight translations from latest Hyrax

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

File: config/locales/blacklight.en.yml [hunk 1]
```diff
@@ -1,3 +1,45 @@
 en:
   blacklight:
     application_name: 'Hyku'
+    search:
```
Comment (jcoyne) [inline config/locales/blacklight.en.yml]: Why don't we let Blacklight provide these?

Comment (jcoyne) [inline config/locales/blacklight.en.yml]: Nm

**Comment by mjgiarlo** [comment]
@jcoyne Isn't Hyku the local app? Where else would you put these in Hyku?

**Comment by jcoyne** [comment]
Sorry, thought this was hyrax.

**Pull request merged**